google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanelTest.java

```diff
@@ -34,11 +36,18 @@
 
   private CloudSdkService cloudSdkService;
 
-  private JLabel warningMessage;
+  private JTextPane warningMessage;
   private TextFieldWithBrowseButton cloudSdkDirectoryField;
 
-  private static final String INVALID_SDK_DIR_WARNING =
-      "No Cloud SDK was found in this directory.";
+  private static final String INVALID_SDK_DIR_WARNING = "<html>\n"
```

Comment:
got it. I wonder if we should look into building an error type in the future.

this is good for now.